src/Datadog.Trace.ClrProfiler.Native/Datadog.Trace.ClrProfiler.Native.vcxproj

```diff
@@ -124,8 +146,17 @@
       <ConformanceMode>true</ConformanceMode>
       <SDLCheck>true</SDLCheck>
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
-      <PreprocessorDefinitions>_TARGET_64BIT;_UNICODE;UNICODE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>SPDLOG_FMT_EXTERNAL;_TARGET_64BIT;BIT64;_UNICODE;UNICODE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
I see you added the missing `BIT64`. Nice catch. Can you add a note in the PR description that this PR also fixes the issue where the profiler log reports x86 even when the process is x64? (only affected `Release` builds). @kevingosse keeps bringing this up 😉

Reply:
copied and pasted 😁